### ISSUE TITLE
Feed Duration Optional - Enhancement Request

### ISSUE THREAD

**Issue by Lockie85** [babybuddy/babybuddy#772]
Hi,

Firstly this application is amazing so thank you to those involved!

I'd like to ask if it might be possible to add a feature where feeding duration is optional. Often we find ourself putting feeds in after the event, sometimes hours after and we're not overly interested in the duration of a feed, only when it started. Much like the the Nappy Change events, which are far simpler to enter.

I know this might appear like an over the top change, but when everything is kicking off at home I can certainly say entering the nappy times is often a lot easier than entering the feed times.

Many Thanks

**Comment by cdubz** [comment]
I think I would be a tough sell on that change but I'll leave this issue open to gauge interest.

**Comment by AstroPhotoJay** [comment]
+1 for this

In Australia, the professional midwifery advice around tracking feedings is that only the start time really matters. This is how they calculate time since last feed, and how many feeds in a day.

Feed duration is only of interest if there are any issues (e.g weight gain too high or low, feeds taking way too long, etc...)

If changing the way databases work is too much, perhaps it would be possible to have BB automatically use the start time if the end time is left blank when entering a feed?

**Comment by dbensmith** [comment]
I agree with this request for the same reasons. We record many feeds retroactively and duration isn't as important as recording a point in time and food volume, so if feed duration were optional in the UI and API, I believe this would improve both UX and WAF.
